a-universe-from-nothing.sh

```diff
@@ -19,7 +31,7 @@ cd config/src/
 
 # Install kayobe.
 cd ~/kayobe
-./dev/install.sh
+./dev/install-dev.sh
```

Comment:
I suppose this is helpful to develop Kayobe, but now the README instructions are different. Let's update the README? The official documentation for installing Kayobe don't use these scripts anyway.

Reply:
Done. Do you think we should do it the 'hard way' and run the full commands? It could take longer, but would be more realistic.

Reply:
Indeed I think it would be much better to use the full commands so there is less magic involved. That's a change for another PR though.